src/scss/elements/_anchor.scss

```diff
@@ -0,0 +1,11 @@
+.anchor-btn{
+    background-color: #337099;
```

Comment:
We should use a variable here which we already have for this colour in the _variables.scss file
```suggestion
    background-color: $astral;
```